go/pserver/etcd_lister.go

```diff
@@ -0,0 +1,120 @@
+package pserver
+
+import (
+	"time"
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/coreos/etcd/clientv3"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	DefaultEtcdTimeout time.Duration = time.Second * time.Duration(5)
```

Comment:
`5 * time.Second` is fine. Constants in Go is very well designed: https://blog.golang.org/constants